filters/auth/oidc.go

```diff
@@ -401,30 +404,46 @@ func getHost(request *http.Request) string {
 	}
 }
 
-func chunkCookie(cookie http.Cookie) (cookies []http.Cookie) {
+func createOidcCookie(name string, value string, maxAge int, domain string) (cookie *http.Cookie) {
```

Comment:
I suggest we make this a method of a filter and derive domain using `extractDomainFromHost(getHost(request), f.subdomainsToRemove))` instead of passing it

Reply:
Fixed that in https://github.com/zalando/skipper/pull/1965/commits/39c16f047e62edf7b25f8690565e2ca46ddda2f4